tests/test_sweet.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from astropy import units as u
+from astropy.utils.data import download_file
+from lightkurve.search import open as lc_read  # Will be read in 2.x
+from numpy.testing import assert_allclose
+
+from exovetter.tce import TCE
+from exovetter.vetters import Sweet
+
+
+@pytest.mark.remote_data
+def test_kepler_1026032():
+    filename = download_file('http://archive.stsci.edu/missions/kepler/lightcurves/0010/001026032/kplr001026032-2009166043257_llc.fits', cache=True)  # noqa
+
+    period = 8.46043892 * u.day
+    epoch_bkjd = 133.7744903
+    duration = 4.73492 * u.hour
+    target_name = "KPLR001026032"
+    event_name = "False Positive"
+
+    tce = TCE(period=period, tzero=epoch_bkjd, duration=duration,
+              target_name=target_name, event_name=event_name)
+
+    lcf = lc_read(filename)
+    lc = lcf.PDCSAP_FLUX
+
+    sweet_vetter = Sweet()
+    sweet_vetter.run(tce, lc)
+
+    res = sweet_vetter.result
+    assert 'OK' in res['msg']
+    assert_allclose(
+        res['amp'], [[5.13524349e+02, 6.28200510e+02, 8.17452934e-01],
```

Comment:
I have no idea what these numbers mean or if they are correct. Each line is supposedly `[amp, amp_unc, amp / amp_unc]` for `0.5 * period`, `period`, and `2 * period`, respectively. Uncertainty seems too high? Please double check. Maybe I made a mistake in the port somewhere. 🤷

Reply:
Without looking at the data, an amplitude uncertainty  of 600pm is plausible.

Reply:
For KIC 1026032, in the DR25 Robovetter evaluation of this signal,  the sweet_amp is 107.03  and sweet_snr is 16.99.  That doesn't appear to agree with the above. Hmm..

Reply:
That's not good. I wonder if the SWEET vetter in DAVE would agree with this PR or the published result.